src/QuDynamics.jl

```diff
@@ -1,6 +1,8 @@
 module QuDynamics
     using QuBase
     using Compat
+    Pkg.checkout("ODE")
```

Comment:
I think it would be better to put this into `.travis.yml` or maybe we can even specify it in `REQUIRE`.